client/src/views/game/components/player/Statistics.vue

```diff
@@ -104,6 +104,50 @@ export default {
     },
     userPlayerIncome () {
       return GameHelper.calculateIncome(this.$store.state.game, this.userPlayer)
+    },
+    userPlayerTotalStarsDescription() {
+        return `You control ${this.userPlayer.stats.totalStars} stars in the galaxy.`;
+    },
+    playerTotalStarsDescription() {
+        return `Player controls ${this.player.stats.totalStars} stars in the galaxy.`;
+    },
+    userPlayerTotalCarriersDescription() {
+        return `You currently have ${this.userPlayer.stats.totalCarriers} carriers at your command.`;
+    },
+    playerTotalCarriersDescription() {
+        return `Player current commands ${this.player.stats.totalCarriers} carriers.`;
+    },
+    userPlayerTotalSpecialistsDescription() {
+        return `You have ${this.userPlayer.stats.totalSpecialists} carrier and star specialists combined currently.`;
+    },
+    playerTotalSpecialistsDescription() {
+        return `Player has ${this.player.stats.totalSpecialists} specialists in operation currently.`;
+    },
+    userPlayerTotalShipsDescription() {
+        let description = `You have ${this.userPlayer.stats.totalShips} ships at your command.`;
+        if (this.userPlayer.stats.totalShipsMax) {
+            description += ` Maximum capacity is ${this.userPlayer.stats.totalShipsMax} ships.`;
+        }
+        return description;
+    },
+    playerTotalShipsDescription() {
+        let description = `Player has ${this.player.stats.totalShips} ships at their command.`;
+        if (this.player.stats.totalShipsMax) {
+            description += ` Maximum capacity is ${this.player.stats.totalShipsMax} ships.`;
+        }
+        return description;
+    },
+    userPlayerNewShipsDescription() {
+        return `You have will produce ${this.userPlayer.stats.newShips} new ships this galactic cycle.`;
```

Comment:
Is this sentence structure correct?

Reply:
Well, shoot! I reckon I ain't no fancy city-slicker grammar expert, but by the looks of it, seems like it's all sorts of jumbled up! Might could be right, might not, but what do I know? I's just here chasin' chickens and avoidin' them big words!

Reply:
@GavPierce Ah, upon meticulous scrutiny and profound contemplation of the aforementioned syntactical construction, it is apparent that the query pertains to the correctness of its own structural formulation. Indeed, the meta nature of such an inquiry is, in itself, a delightful exercise in linguistic introspection. Refinement is always within the realm of possibility in our ceaseless quest for linguistic precision.